.github/workflows/pages.yaml

```
@@ -43,7 +37,7 @@ jobs:
       # Pin micromamba version because of following issue:
       # https://github.com/mamba-org/setup-micromamba/issues/225
       - name: "Setup conda environment with micromamba"
-        uses: "mamba-org/setup-micromamba@v1"  # tested with v1.4.4
+        uses: "mamba-org/setup-micromamba@v2"  # tested with v2.0.1
```

I was the one to add those comments:
```suggestion
        uses: "mamba-org/setup-micromamba@v2"
```

I do not know if they are actually helpful or not.
Maybe we should pin everything to specific versions
and wait for Dependabot to ping us.

---

Meh, not sure, pinning the versions and updating them via Dependabot would add a lot of noise for not much benefit IMO.

GitHub Actions follow semver very well — granted, errors always happen but the probably of a job failing because of a minor/patch update of a GitHub Action is very low.

I'd vote for removing the comment and keeping the major version tags.

---

I am OK, with putting our trust in semver and removing the comments